app/controllers/hyrax/user_pinned_collections_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module Hyrax
+  class UserPinnedCollectionsController < ApplicationController
```

Comment:
how does the `UserPinnedCollections` feature relate to the analytics one?

it seems nice, but just seems like a very different feature set.

Reply:
Agreed. I removed this feature and I will put this in a separate PR.